
"""Defines error handling functions.
"""

def exception(msg):
  print(msg)
  exit(0)

def error(msg):
  print("Error: " + msg)
  exit(1)
